### PR TITLE
Microservices assignment

### PR DESCRIPTION
**Pull request by germanleonz**

### CODE REVIEW

File: src/main/java/com/tenx/ms/retail/stock/repository/StockRepository.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+import com.tenx.ms.retail.stock.domain.StockEntity;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.Optional;
+
+
+@Repository
+public interface StockRepository extends JpaRepository<StockEntity, Long>{
+    Optional<StockEntity> findOneByStore_StoreIdAndProductId(Long storeId, Long productId);
```
Comment (shuangli) [inline src/main/java/com/tenx/ms/retail/stock/repository/StockRepository.java]: It is not recommended to use _ in method name. for this case, you can change it to findOneByStoreIdAndProductId

**Comment by shuangli** [comment]
please remove lib/spring-data-jpa-1.3.3.jar

File: src/main/java/com/tenx/ms/retail/stock/repository/StockRepository.java (hunk 1 above)
Comment (germanleonz) [reply src/main/java/com/tenx/ms/retail/stock/repository/StockRepository.java]: @shuangli thanks. Fixed.

**Comment by shuangli** [comment]
where are your test cases?

**Comment by germanleonz** [comment]
@shuangli I'm currently working on them. I'll upload them tomorrow.

File: src/main/java/com/tenx/ms/retail/order/domain/OrderEntity.java [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+    @Column(name="order_id", nullable = false)
+    private Long orderId;
+
+    @Column(name="status", nullable = false, length = 7)
+    private String status;
+
+    @Column(name = "order_date", columnDefinition = "datetime default current_timestamp",
+            insertable = false, nullable = false)
+    @Temporal(TemporalType.TIMESTAMP)
+    private Date orderDate;
```
Comment (wingng) [inline src/main/java/com/tenx/ms/retail/order/domain/OrderEntity.java:34]: You can use LocalDateTime from Java 8. It's better version of Date class

**Pull request closed without merging**